pages/callbacks/overview_callbacks.py

```diff
@@ -339,30 +331,18 @@ def contributor_growth_line_bar(df_contrib):
     fig.update_layout(
         xaxis=dict(
             rangeselector=dict(
-                buttons=list([
-                    dict(count=1,
-                        label="1m",
-                        step="month",
-                        stepmode="backward"),
-                    dict(count=6,
-                        label="6m",
-                        step="month",
-                        stepmode="backward"),
-                    dict(count=1,
-                        label="YTD",
-                        step="year",
-                        stepmode="todate"),
-                    dict(count=1,
-                        label="1y",
-                        step="year",
-                        stepmode="backward"),
-                    dict(step="all")
-                ])
-            ),
-            rangeslider=dict(
-                visible=True
+                buttons=list(
+                    [
```

Comment:
same as above

Reply:
It's because of the Black formatter that Michael Clifford suggested.